packages/ui/src/views/repo/components/token-dialog/clone-credential-dialog.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect } from 'react'
+import { useForm } from 'react-hook-form'
+
+import {
+  Button,
+  ButtonGroup,
+  CopyButton,
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  Input
+} from '@/components'
+import { TranslationStore } from '@/views'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { z } from 'zod'
+
+interface CloneCredentialDialogProps {
+  open: boolean
+  onClose: () => void
+  onManageToken: () => void
+  useTranslationStore: () => TranslationStore
+  tokenData: {
+    identifier: string
+    lifetime: string
+    token: string
+  }
+}
+const formSchema = z.object({
+  identifier: z.string(),
+  lifetime: z.string(),
+  token: z.string()
+})
+
+export type TCloneCredentialsDialog = z.infer<typeof formSchema>
+
+export const CloneCredentialDialog: React.FC<CloneCredentialDialogProps> = ({
+  open,
+  onClose,
+  onManageToken,
+  tokenData,
+  useTranslationStore
+}) => {
+  const { t } = useTranslationStore()
+  const { setValue } = useForm<TCloneCredentialsDialog>({
+    resolver: zodResolver(formSchema),
+    defaultValues: tokenData
+  })
+  useEffect(() => {
+    if (tokenData) {
+      setValue('identifier', tokenData.identifier)
+      setValue('lifetime', tokenData.lifetime)
+      setValue('token', tokenData.token)
```

Comment:
why is this needed when `tokenData` is already being passed to `defaultValues` above?

Reply:
yeah, not needed, will remove this